Client/ClientGame.js

```diff
@@ -70,13 +68,12 @@ var Game = function (fps) {
     // create a ground
     var ground = Bodies.rectangle(400, 610, 810, 60, { isStatic: true });
     
-    //TODO(Networking): implement in Server Game 
+    //TODO: What is this item? 
```

Comment:
The small block that spawns is that item. Its there for testing and can be removed